src/octoprint/plugins/softwareupdate/static/clientjs/softwareupdate.js

```diff
@@ -13,6 +13,7 @@
         this.updateUrl = url + "update";
         this.configureUrl = url + "configure";
         this.updatelogUrl = url + "updatelog";
+        this.cancelQueuedURL = url + "cancel_queued";
```

Comment:
If adjusted as suggested above, this would turn into

```suggestion
        this.queuedURL = this.updateUrl + "/queued";
```